.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-dist: xenial
+dist: bionic
```

Comment:
why changing it?

Reply:
since ppc64le on distribution xenial is not supported OpenSSL ,i have modified supporting version of dist - bionic .
If you wish to apply only for ppc64le then i can go ahead and modify script to include only for ppc64le,Please let me know.

Reply:
i think it's fine, thanks.